extensions/wikia/MercuryApi/MercuryApiHooks.class.php

```diff
@@ -4,6 +4,7 @@ class MercuryApiHooks {
 
 	const SERVICE_API_BASE = '/api/v1/';
 	const SERVICE_API_ARTICLE = 'article/';
+	const SERVICE_API_CURATED_CONTENT = 'curatedContent/';
```

Comment:
It's `/main/section/{sectionName}` and `/main/category/{categoryName}` (https://github.com/Wikia/mercury/blob/dev/server/routes.ts#L105-L114).

We should purge the category URL too, right?